packages/pill/src/__tests__/Pill.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Pill from '../Pill';
+
+const dataTestId = 'test-id';
+const defaultChild = 'text';
+
+describe('Pill', () => {
+  /**
+   * Attributes tests
+   */
+  describe('Attributes tests', () => {
+    it('should set `data-test-id` attribute', () => {
+      const { getByTestId } = render(
+        <Pill dataTestId={dataTestId}>{defaultChild} </Pill>,
+      );
+
+      expect(getByTestId(dataTestId).tagName).toBe('DIV');
+    });
+
+    it('should set text', () => {
+      const { container } = render(<Pill>{defaultChild}</Pill>);
+
+      expect(container.firstElementChild).toHaveTextContent(defaultChild);
+    });
+
+    it('should set leftIcon', () => {
+      const { container } = render(
+        <Pill leftIcon={<span>Left</span>}>{defaultChild}</Pill>,
```

Comment:
you can take any sag icon from the icons package

Reply:
resolved